packages/waveform-progress/src/waveform-progress.ts

```diff
@@ -115,7 +115,7 @@ export default class WaveformProgress extends LitElement {
 
   static get styles() {
     const fillColorCss = css`var(--fillColor, #3272b6)`;
-    const zoneOfSilenceColorCss = css`var(--zoneOfSilenceColor, orange)`;
+    const zoneOfSilenceColorCss = css`var(--zoneOfSilenceColor, #f6e652)`;
```

Comment:
can we make this a variable?

Reply:
This is already a variable, I just changed the default value for it to match the one from the mocks. You can override like this:
```
<style>
  waveform-progress {
    --zoneOfSilenceColor: purple;
  }
</style>
```